### PR TITLE
fix syntax for test/README.md

### PR DESCRIPTION
**Pull request by mdshuai**
fix syntax format error for test/README.md

Signed-off-by: DeShuai Ma dma@redhat.com

### CODE REVIEW

File: test/README.md [hunk 1]
```diff
@@ -78,8 +78,7 @@ make localintegration RUNTIME=cc-oci-runtime
 
 ## Writing integration tests
 
-[Helper functions]
-(https://github.com/kubernetes-incubator/crio/blob/master/test/helpers.bash)
+[Helper functions](https://github.com/kubernetes-incubator/crio/blob/master/test/helpers.bash)
```
Comment (runcom) [inline test/README.md]: there is another typo actually: `crio -> cri-o`

Comment (mdshuai) [reply test/README.md]: done

**Comment by runcom** [comment]
LGTM let's just wait for travis and merge /cc @mrunalp

**Comment by TomSweeneyRedHat** [comment]
@mdshuai Thanks for the change and it LGTM, but you need to sign your commit.

**Comment by mdshuai** [comment]
@TomSweeneyRedHat done

**Comment by runcom** [comment]
Your commit isn't signed yet and there's no need to run all the tests for this change :)

**Comment by mdshuai** [comment]
should ready now. thanks

**Comment by TomSweeneyRedHat** [comment]
Thank you @mdshuai!

**Pull request merged**